pyquil/api/_quantum_computer.py

```diff
@@ -94,10 +105,14 @@ def __init__(self, *, name: str, qam: QAM, device: AbstractDevice, compiler: Abs
 
         self.symmetrize_readout = symmetrize_readout
 
-    def qubit_topology(self):
+    def qubits(self) -> List[int]:
```

Comment:
im confused why a lot of these changes are getting picked up

Reply:
I somehow messed up my local git. Going to nuke this pull request and try again.